### DISCUSSION TITLE
Disable a destination in a channel?

### DISCUSSION THREAD

**Discussion by wintertime12** [nextgenhealthcare/connect#4926] [Q&A] [answered]
Hello all,

I have a channel on MIRTH with a source and five destinations. One of the destinations sends HL7 messages to a downstream system. This downstream system was discontinued and the system was shut down.  Now MIRTH has queued messages for this system. I want to let this destination stop working, but still want to keep the destination setup and the filter etc.

Can I just disable this destination and deploy the channel. 

Thank you,

**Comment by jonbartels** [comment, marked as answer]
Disabling the destination will preserve its settings (filters, transformers, etc) but not actually run the destination.

**Reply by wintertime12** [reply]
Thanks so much jonbartels!

Once I disable the destination, what will happen to those queued messages. Will those queued messages cause any troubles?

**Reply by jonbartels** [reply]
If you need the queued messages - you should export them from MC admin.
If you do not need the queued messages - you should remove them before disabling your destination

Disabling the destination *should* retain the queued messages. This could be good in that they are saved, but they will never be sent and will never be pruned.

I do not remember what impact this will have on your channels message statistics.
